demisto_sdk/commands/init/tests/initiator_test.py

```diff
@@ -470,7 +473,67 @@ def test_template_integration_init(initiator, tmpdir, template):
     assert not diff, f'There\'s a missing file in the copied files, diff is {diff}'
 
 
-def test_script_init(initiator, tmpdir):
+def test_template_integration_init_with_ignore_secrets(initiator, tmpdir, mocker):
+    """
+    Tests `integration_init` function with a given integration template name.
+
+    Given
+        - Inputs to init integration in a given output.
+        - An integration template - HelloWorld.
+
+    When
+        - Running the init command with secrete ignore.
+
+    Then
+        - Ensure the function's return value is True
+        - Ensure integration directory with the desired integration name is created successfully.
+        - Ensure integration directory contains all the files of the template integration.
+        - Ensure .secrets-ignore file is not empty.
+    """
+
+    white_list = {
+        "files": [],
+        "iocs": {},
+        "generic_strings": []
+    }
+    temp_pack_dir = os.path.join(tmpdir, PACK_NAME)
+    os.makedirs(temp_pack_dir, exist_ok=True)
+    os.makedirs('Tests', exist_ok=True)
+    os.makedirs('Packs/PackName', exist_ok=True)
+    with open('Tests/secrets_white_list.json', 'w') as f:
+        json.dump(white_list, f)
+
+    secrets_ignore_path = 'Packs/PackName/.secrets-ignore'
+
+    initiator.output = temp_pack_dir
+    initiator.dir_name = INTEGRATION_NAME
+    initiator.is_integration = True
+    initiator.template = 'HelloWorld'
+    initiator.category = 'Utilities'
+
+    integration_path = os.path.join(temp_pack_dir, INTEGRATION_NAME)
+    mocker.patch('builtins.input', return_value='y')
+    mocker.patch('demisto_sdk.commands.init.initiator.get_pack_name', return_value='PackName')
+
+    res = initiator.integration_init()
+
+    integration_dir_files = set(listdir(integration_path))
+    expected_files = {
+        "Pipfile", "Pipfile.lock", "README.md", f"{INTEGRATION_NAME}.py",
+        f"{INTEGRATION_NAME}.yml", f"{INTEGRATION_NAME}_description.md", f"{INTEGRATION_NAME}_test.py",
+        f"{INTEGRATION_NAME}_image.png", "test_data", "command_examples"
+    }
+
+    assert res
+    assert os.path.isdir(integration_path)
+    diff = expected_files.difference(integration_dir_files)
+    assert not diff, f'There\'s a missing file in the copied files, diff is {diff}'
```

Comment:
Not sure this error is informative enough, what copied files? do you mean the files created by the init command?

Reply:
I dont see this has changed.
Also, I'm not sure when the user will get here.
They are just running the command, why would there be missing files?